### DISCUSSION TITLE
ERROR [root] Error: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

### DISCUSSION THREAD

**Discussion by qinlz-1** [apache/superset#18346] [Q&A / Help]
### Expected results

what you expected to happen.

### Actual results

what actually happens.

### Environment

(please complete the following information):

- browser type and version:
- superset version: `superset version`
- python version: `python --version`
- node.js version: `node -v`
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

superset upgrade from version 0.37.2

superset db upgrade
logging was configured successfully
INFO:superset.utils.logging_configurator:logging was configured successfully
/usr/local/py3env/lib/python3.6/site-packages/flask_caching/__init__.py:192: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.
  "Flask-Caching: CACHE_TYPE is set to null, "
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
ERROR [root] Error: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

**Comment by villebro** [comment]
@qinlz-1 can you specify which version you are trying to upgrade to? One quick hack you may want to try is upgrading to 0.38.x, then 1.0.x, then 1.1.x etc, and see if that works. Also, Superset no longer supports Python 3.6, so you will need to upgrade to at least 3.7 if you want to run one of the latest versions.

**Comment by kopsteve** [comment]
superset upgrade，Python 3.9.19 Superset 4.0.1 node v22.2.0
